src/containers/Storage/Pdisk/Pdisk.tsx

```diff
@@ -42,6 +42,9 @@ const colorSeverity = {
     Orange: 4,
     Red: 5,
 };
+
+const NOT_AVAILABLE_SEVERITY = colorSeverity.Red;
```

Comment:
Let's decouple color constants in Pdisk and Vdisk files into separate file "helpers" or "constants".